pygpu/tests/main.py

```diff
@@ -1,8 +1,9 @@
 import os
+import nose.plugins.builtin
```

Comment:
We do use nose.plugins.builtin in the code below, and I fixed the references so that flake8 will no longer complain, so I don't think a comment is necessary.

We might want to add a test of pygpu.test() to the buildbot though.